examples/gpx/ULTRA2009.gpx

```diff
@@ -0,0 +1,10683 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why not using https://github.com/iTowns/iTowns2-sample-data ? This repo was meant for that right?